.github/workflows/build.yml

```
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['2.5', '2.6', '2.7']
+        ruby: ['2.5', '2.6', '2.7', '3.0.1']
```

As we rely on the latest minor version for the former ruby versions here, I'd vote for doing the same on the 3.x release. 

```suggestion
        ruby: ['2.5', '2.6', '2.7', '3.0']
```